query-node/mappings/src/bounty.ts

```diff
@@ -50,13 +50,12 @@ async function getBounty(store: DatabaseManager, bountyId: BountyId | string, re
   return bounty
 }
 
-async function getContribution(
+function getContribution(
   store: DatabaseManager,
   bountyId: BountyId,
   contributorId: string | undefined
 ): Promise<BountyContribution | undefined> {
-  const contributor = whenDef(contributorId, (id) => ({ contributor: id })) ?? null
-  return await store.get(BountyContribution, { where: { bountyId, contributor } })
+  return store.get(BountyContribution, { where: { bountyId, contributor: { id: contributorId ?? null } } })
```

Comment:
```suggestion
  return store.get(BountyContribution, { where: { bounty: { id: bountyId }, contributor: { id: contributorId ?? null } } })
```

Reply:
Sorry I just realized I forgot about the bounty. But please check my reply above :point_up: